dev-docs/bidders/bizzclick.md

```diff
@@ -12,17 +12,35 @@ safeframes_ok: true
 deals_supported: true
 pbjs: true
 pbs: true
+pbs_app_supported: true
 sidebarType: 1
+floors_supported: true
+prebid_member: false
+fpd_supported: false
+gvl_id: none
+multiformat_supported: will-bid-on-one
+ortb_blocking_supported: true
+userIds: all
 ---
 
 ### Note
 
 The Example Bidding adapter requires setup before beginning. Please contact us at <support@bizzclick.com> .BizzClick will only respond to the first impression and that multiple ad formats of that single impression are not supported.
 
-### Bid Params
+### Bid Params for Prebid Server and Prebid Mobile
 
 {: .table .table-bordered .table-striped }
-| Name          | Scope    | Description           | Example   | Type      |
+| Name | Scope | Description | Example | Type |
 |---------------|----------|-----------------------|-----------|-----------|
-| `placementId`      | required | placement id | `'hash'`    | `string` |
-| `accountId`      | required | account id | `'bizzclickTest'`    | `string` |
+| `sourceId` | required | Unique hash provided by bizzclick | `'6dllcEHSxYdSb6yLmCqE'` | `string` |
+| `accountId` | required | Unique name provided by bizzclick | `'bizzclick-test'` | `string` |
+| `host` | optional | Bizzclick server region. US East by default | `'us-e-node1'` | `string` |
```

Comment:
Asking for publishers to enter the geographic region is not the right way to do this.

For Prebid.js, you really ought to be paying for a global traffic management solution.
For Prebid Server, a solution is to have the host company deploy a different endpoint in different regions. See https://github.com/prebid/prebid-server/pull/3311

Since the host parameter is optional, I won't insist, but I'm going to request that you consider how difficult it might be for a publisher to carry out this request.